pyproject.toml

```diff
@@ -33,3 +33,35 @@ markers = [
     "asyncio: tests that require an asyncio eventloop",
 ]
 junit_family = "xunit1"
+
+[tool.liccheck]
+authorized_licenses = [
+	"apache software",
+	"public domain",
+	"python software foundation",
+	"apache license 2.0",
+	"mit license",
+	"mozilla public license 2.0 (mpl 2.0)",
+	"zope public license",
+	"zope public",
+	"bsd 3-clause",
+	"isc license (iscl)",
+	"bsd",
+	"3-clause bsd license",
+	"3-clause bsd",
+	"apache 2.0",
+	"mit",
+	"historical permission notice and disclaimer (hpnd)",
+]
+unauthorized_licenses = [
+	"gnu lesser general public license v3 or later (lgplv3+)",
```

Comment:
IANAL, but why forbid this here and allow it below for `mirakuru`?

In the LC LGPL3 is accepted as well: https://github.com/raiden-network/light-client/blob/master/.github/workflows/workflow.yml#L45

Reply:
The reasoning here is that we have to include additional notes of a package in the `LICENSE` file for each 
LGPL dependency. If this would pass automatically, the developer introducing such dependencies will not notice that actions should be taken.
It would be best to document this somewhere though

Reply:
Maybe just add a comment above the exceptions on what disclaimer to add when adding an exception.

Reply:
Added comments in the pyproject.toml